tsconfig.json

```diff
@@ -27,7 +29,8 @@
   },
   "lib": ["esnext"],
   "include": [
-    "./src/**/*.ts"
+    "./src/**/*.ts",
+    "./test/**/*.ts"
```

Comment:
this addition, as we saw, is causing the `dist/` folder to now contain `/src` and `/test` sub folders.  Which means the service won't currently start.

Reply:
Thanks, i'll fix this. There is a suitable solution here: https://stackoverflow.com/a/60223923